<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>common</class>
 <widget class="QWidget" name="common">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background-color:rgb(0, 10, 117);
	color:rgb(255,255,255);
}

QGroupBox{
	border:none;
}
QScrollArea{
	border:none;
}
QRadioButton::indicator{
	width:15px;
	height:15px;
	border-radius:8px;
	border-style: solid;
	border-width: 1px;
	border-color: black;
}
QCheckBox::indicator:unchecked{
	background-color:rgb(255, 255, 255);
}
QCheckBox::indicator:checked {
	background-color:rgb(255, 170, 127);
}
QRadioButton::indicator:checked {
	background-color:rgb(255, 170, 127);
}
QRadioButton::indicator:unchecked {
	background-color:rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>360</width>
        <height>1188</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="channel">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item alignment="Qt::AlignLeft|Qt::AlignTop">
             <widget class="QLabel" name="label_4">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>channel:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QGroupBox" name="channel1to64">
              <property name="maximumSize">
               <size>
                <width>166</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="title">
               <string/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="spacing">
                <number>10</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QCheckBox" name="c1t8">
                 <property name="text">
                  <string>channel 1-8</string>
                 </property>
                 <property name="autoExclusive">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="c9t16">
                 <property name="text">
                  <string>channel 9-16</string>
                 </property>
                 <property name="autoExclusive">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="c17t24">
                 <property name="text">
                  <string>channel 17-24</string>
                 </property>
                 <property name="autoExclusive">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="c25t32">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>channel 25-32</string>
                 </property>
                 <property name="autoExclusive">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="c33t40">
                 <property name="text">
                  <string>channel 33-40</string>
                 </property>
                 <property name="autoExclusive">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="c41t48">
                 <property name="text">
                  <string>channel 41-48</string>
                 </property>
                 <property name="autoExclusive">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="c49t56">
                 <property name="text">
                  <string>channel 49-56</string>
                 </property>
                 <property name="autoExclusive">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="c57t64">
                 <property name="text">
                  <string>channel 57-64</string>
                 </property>
                 <property name="autoExclusive">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft">
             <widget class="QGroupBox" name="channelselect">
              <property name="title">
               <string/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="spacing">
                <number>10</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QCheckBox" name="c1">
                 <property name="text">
                  <string>channel 1</string>
                 </property>
                 <property name="tristate">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="c2">
                 <property name="text">
                  <string>channel 2</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="c3">
                 <property name="text">
                  <string>channel 3</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="c4">
                 <property name="text">
                  <string>channel 4</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="c5">
                 <property name="text">
                  <string>channel 5</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="c6">
                 <property name="text">
                  <string>channel 6</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="c7">
                 <property name="text">
                  <string>channel 7</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="c8">
                 <property name="text">
                  <string>channel 8</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="maxfrequency">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>6</number>
            </property>
            <item alignment="Qt::AlignLeft|Qt::AlignTop">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>maxfrequency:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_16" stretch="0">
            <item>
             <widget class="QGroupBox" name="maxfrequency_box">
              <property name="title">
               <string/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="spacing">
                <number>10</number>
               </property>
               <item>
                <widget class="QRadioButton" name="h20">
                 <property name="text">
                  <string>20Hz</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="h40">
                 <property name="text">
                  <string>40Hz</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="h60">
                 <property name="text">
                  <string>60Hz</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="h100">
                 <property name="text">
                  <string>100Hz</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="h200">
                 <property name="text">
                  <string>200Hz</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="h500">
                 <property name="text">
                  <string>500Hz</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="model">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item alignment="Qt::AlignLeft|Qt::AlignTop">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>model:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_17" stretch="2">
            <item>
             <widget class="QGroupBox" name="modelbox">
              <property name="title">
               <string/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="spacing">
                <number>10</number>
               </property>
               <item>
                <widget class="QRadioButton" name="line">
                 <property name="text">
                  <string>line</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="log">
                 <property name="text">
                  <string>log</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="window">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_18" stretch="0,2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item alignment="Qt::AlignLeft|Qt::AlignTop">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>window:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QGroupBox" name="windowbox">
              <property name="title">
               <string/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>10</number>
               </property>
               <item>
                <widget class="QRadioButton" name="min1">
                 <property name="text">
                  <string>1 min</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="min3">
                 <property name="text">
                  <string>3 min</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="min6">
                 <property name="text">
                  <string>6 min</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="min15">
                 <property name="text">
                  <string>15 min</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="min30">
                 <property name="text">
                  <string>30 min</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="timelength">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>timelength:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_19" stretch="2">
            <item>
             <widget class="QGroupBox" name="timelengthbox">
              <property name="title">
               <string/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="spacing">
                <number>10</number>
               </property>
               <item>
                <widget class="QRadioButton" name="s1">
                 <property name="text">
                  <string>1 s</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="s3">
                 <property name="text">
                  <string>3 s</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="s5">
                 <property name="text">
                  <string>5 s</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="s10">
                 <property name="text">
                  <string>10 s</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="s20">
                 <property name="text">
                  <string>20 s</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="scale">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item alignment="Qt::AlignLeft|Qt::AlignTop">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>scale:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_20" stretch="2">
            <item>
             <widget class="QGroupBox" name="scalebox">
              <property name="title">
               <string/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <property name="spacing">
                <number>10</number>
               </property>
               <item>
                <widget class="QRadioButton" name="uvauto">
                 <property name="text">
                  <string>auto</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="uv50">
                 <property name="text">
                  <string>50 uv</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="uv100">
                 <property name="text">
                  <string>100 uv</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="uv200">
                 <property name="text">
                  <string>200 uv</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="uv1000">
                 <property name="text">
                  <string>1000 uv</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="uv10000">
                 <property name="text">
                  <string>10000 uv</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>43</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(61, 127, 255);</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="done">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>43</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(48, 171, 195);</string>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
